src/Microsoft.VisualStudio.SlowCheetah/Build/Microsoft.VisualStudio.SlowCheetah.targets

```diff
@@ -58,18 +66,41 @@ Copyright (C) Sayed Ibrahim Hashimi, Chuck England 2011-2013. All rights reserve
     <Message Text="SlowCheetah: Collecting Transform files" Importance="low"/>
 
     <ItemGroup>
-      <ScFilesToTransform Include="@(None);@(Content);@(Resource);@(EmbeddedResource)"
-                         Condition=" '%(TransformOnBuild)' == 'true' ">
+      <_ScNoneFilesToTransform Include="@(_NoneWithTargetPath)" Condition="'%(TransformOnBuild)' == 'true'">
```

Comment:
For all of these item types, default `CopyToOutputDirectory` to `PreserveNewest` if it is not set.

``` xml
<CopyToOutputDirectory Condition=" '$(CopyToOutputDirectory)' == '' ">PreserveNewest</CopyToOutputDirectory>
```